Advocatus/settings.py

```diff
@@ -89,7 +89,7 @@
     DEBUG = True
     # SECURITY WARNING: keep the secret key used in production secret!
     SECRET_KEY = 'django-insecure-f*$r=69zkt46r571$gvy%tx9!4#rps&v64a=qz9*+_5ajit64o'
-    ALLOWED_HOSTS = []
+    ALLOWED_HOSTS = ['http://advocatus-gjfxgcdhgbd8d4a3.brazilsouth-01.azurewebsites.net/']
```

Comment:
```suggestion
    ALLOWED_HOSTS = ['advocatus-gjfxgcdhgbd8d4a3.brazilsouth-01.azurewebsites.net']
```

Reply:
The protocols and paths shouldn't be included

Reply:
ok